lib/stripe/credit_card_clone_finder.rb

```diff
@@ -8,7 +8,8 @@ def initialize(card, stripe_account)
     end
 
     def find_cloned_card
-      return nil unless fingerprint = fingerprint_for_card(@card) && email = @card.user&.email
```

Comment:
Yeah, assignments are usually discouraged in conditionals. Combining that with boolean logic without brackets isn't very clear.